openshift/buildConfig-template.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
     description: >
       This is Osiris API BuildConfig Template.
     openshift.io/display-name: 'Thoth: Osiris API - BuildConfig'
+    version: 0.1.0
```

Comment:
What this version corresponds to? Is it a version of the template itself? (same for ConfigMap template)

Reply:
ja

Reply:
Cool, thanks!